lisa/tools/lspci.py

```diff
@@ -36,6 +36,13 @@
     re.MULTILINE,
 )
 
+# With -mnn option, result would be with vendor/device id
+# d8:00.0 "Ethernet controller [0200]" "Mellanox Technologies [15b3]"
+#   "MT27520 Family [ConnectX-3 Pro] [1007]" "Mellanox Technologies [15b3]"
+#   "Mellanox Technologies ConnectX-3 Pro Stand-up dual-port 40GbE MCX314A-BCCT [0006]"
+PATTERN_DEVICE_ID = re.compile(r"\[([^\]]{4})\]")
```

Comment:
Use named pattern, it makes the code easier to read.

Reply:
Sure Chi, will use `PATTERN_DEVICE_ID = re.compile(r"\[(?P<id>[^\]]{4})\]")`

This is to fetch vendor/device id from vendor/device_info.

Reply:
@squirrelsc Please check the latest push

Reply:
@squirrelsc Used method provided by `lisa.utils` now. Please check the latest push